next/components/forms/simple-components/ServiceCard.tsx

```diff
@@ -0,0 +1,49 @@
+import cx from 'classnames'
+import { FunctionComponent, HTMLAttributeAnchorTarget, SVGProps } from 'react'
+
+import CircleArrowRight from '../../../assets/images/circle-arrow-right.svg'
+
+type ServiceCardBase = {
+  title: string
+  description: string
+  buttonText: string
+  Logo: FunctionComponent<SVGProps<SVGSVGElement>>
+  href: string
+  target?: HTMLAttributeAnchorTarget | undefined
+  className?: string
+}
+
+const ServiceCard = ({
+  title,
+  description,
+  buttonText,
+  Logo,
+  href,
+  target,
+  className,
+}: ServiceCardBase) => {
+  const style = cx(
+    'w-[280px] bg-gray-0 border-gray-200 flex flex-col items-start p-5 gap-5 border-solid border-2 rounded-lg',
+    className,
+  )
+
+  return (
+    <a href={href} target={target} rel="noreferrer">
```

Comment:
this is a visual component , it shouldn't contain the linking behaviour - often you'll be wrapping this in Next's Link https://nextjs.org/docs/api-reference/next/link because you'll be linking within the same application. Please remove.